nested/controller.json

```diff
@@ -0,0 +1,234 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "moodleCommon": {
+            "metadata": {
+                "description": "Common Moodle values"
+            },
+            "type": "object"
+        },
+        "blobStorageAccountKey": {
+            "metadata": {
+                "description": "Key used to create a storage container"
+            },
+            "type": "string"
+        },
+        "redisPrimaryKey": {
+            "metadata": {
+                "description": "Key used for Redis authentication"
+            },
+            "type": "string"
+        }
+    },
+    "resources": [
+        {
+            "apiVersion": "2015-06-15",
```

Comment:
update apiVersion - do this through all the files, make sure none of the apiVersions are more than 1 year old

Reply:
Done